projects/uswds-components/src/lib/form-field/form-field.component.html

```diff
@@ -0,0 +1,14 @@
+<div #labelDiv [class.usa-form-group--error]="!!errorMessage" class="usa-form-group ">
+  <label *ngIf="!!label" [attr.for]="id" class="usa-label" [class.usa-input-error-label]="errorMessage">
+    {{ label || '' }}
+    <span *ngIf="required || requiredFlag">
+      (Required)
+    </span>
+  </label>
+  <div *ngIf="description" [attr.id]="id" class="usa-label--description form-text text-muted" [innerHTML]="description">
```

Comment:
let's use usa-hint here rather than sam-styles classes

Reply:
Updated the tag with usa-hint class